linuxgsm.sh

```diff
@@ -20,7 +20,7 @@ if [ -f ".dev-debug" ]; then
 	set -x
 fi
 
-version="v19.12.4"
+version="v19.12.5"
```

Comment:
the version number should not be included in the commit.
It looks like you need to merge the latest develop branch into your branch